src/components/member/BusinessDetails/BusinessDetails.jsx

```diff
@@ -36,18 +37,12 @@ const BusinessDetailsPicture = (props) => {
   } = props;
   return (
     <Container className="business-details-picture__container">
-      <div className="business-details-picture__content">
-        <BusinessProfilePicture
-          imageUrl={imageUrl}
-          onClickBusinessProfileImage={() => {
-            props.onClickBusinessProfileImage({
-              type: MEMBER_DETAILS_TYPE.BUSINESS,
-              id,
-            });
-          }}
-        />
-        <div className="business-details__category">{businessCategory}</div>
-      </div>
+      <BusinessProfilePicture

```

Comment:
This is the correct Component as per latest changes.
```suggestion
      <BusinessProfilePicture
          imageUrl={imageUrl}
          onClickBusinessProfileImage={() => {
            props.onClickBusinessProfileImage({
              type: MEMBER_DETAILS_TYPE.BUSINESS,
              id,
            });
          }}
        />
```

Reply:
Done